server/market/market.go

```diff
@@ -378,7 +395,10 @@ func (m *Market) runEpochs(nextEpochIdx int64) {
 
 			// Stamp and process the order in the target epoch queue.
 			s.rec.order.SetTime(sTime)
-			m.processOrder(s.rec, orderEpoch, s.errChan)
+			err := m.processOrder(ctx, s.rec, orderEpoch, s.errChan)
+			if err != nil {
+				return
+			}
```

Comment:
Does this error need to be logged?

Reply:
Good catch.  I had initially thought about returning an error from Run.